src/kit/Theme/UI.tsx

```diff
@@ -192,7 +192,9 @@ export const UI: React.FC<{
 
   return (
     <div className={className}>
-      <ConfigProvider theme={configTheme}>{children}</ConfigProvider>
+      <App>
+        <ConfigProvider theme={configTheme}>{children}</ConfigProvider>
+      </App>
```

Comment:
swap how these are wrapped -- the antd docs indicate that the app component uses the config provider for tokens.